lection1/arrToHash.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+// 2.Преобразуйте массив в объект используя функцию reduce.
+
+const videos = [
+  {
+    id: 65432445,
+    title: "The Chamber",
+  },
+  {
+    id: 675465,
+    title: "Fracture",
+  },
+  {
+    id: 70111470,
+    title: "Die Hard",
+  },
+  {
+    id: 654356453,
+    title: "Bad Boys",
+  },
+];
+
+function arrToHash(arr) {
+  return arr.reduce((acc, el) => {
```

Comment:
don't shortcut parameter's names. you iterate throw videos --> element at iteration - video